master/security/kubernetes-network-policy.md

```diff
@@ -4,60 +4,60 @@ title: Get started with Kubernetes network policy
 
 ### Big picture
 
-Kubernetes network policy lets administrators and developers enforce which network connections are allowed using rules.
+Kubernetes network policy lets administrators and developers enforce which network traffic is allowed using rules.
```

Comment:
I think connections is more accurate - network policy is enforced on new connections, not on all packets that are flowing.

Reply:
We can still choose to switch to traffic if it makes more sense for a reader, but at an implementation level it's connection-oriented rather than packet-oriented.

Reply:
Interesting...we switched to "connection" language during our first iteration of How to template. Then, I saw how "traffic" was used pervasively throughout both doc sets...it would be enormous effort to align all of the docs, so I switched back to traffic. It seemed worse to have some "connection" and some "traffic." I prefer connection as well.

Reply:
Let's make this a backlog item.